src/cloud/social/alias/randombytes.js

```diff
@@ -1,33 +1,55 @@
 // Alternative to crypto-browserify's randombytes shim which
-// falls back to Math.random when crypto.getRandomValues is
+// falls back to freedom.js core.crypto when crypto.getRandomValues is
 // unavailable, which is currently the case in Firefox web workers.
-// TODO: Fallback to something better such as freedomjs' core.crypto
-//       module (difficult because it is an asynchronous API).
+// TODO: post FF48 (2016-08-02) switch to just native crypto.getRandomValues
```

Comment:
You're right, it should - my thought was more to just remove the fill at that point since it won't be used in any platform and will just be ambiguous dead code. I'll clarify the comment.